assets/js/session/index.js

```diff
@@ -274,6 +274,10 @@ function handleDocumentKeyDown(hook, event) {
     if (cmd && key === "s") {
       cancelEvent(event);
       saveNotebook(hook);
+    } else if (keyBuffer.tryMatch(["S", "n"])) {
+      insertSectionBelowFocusedCell(hook);
+    } else if (keyBuffer.tryMatch(["S", "N"])) {
```

Comment:
My reasoning for these is `S` like a section prefix and then `n`/`N` as new, similarly to inserting an Elixir cell. Lower case `S` would be nicer, but we already use it for *showing* stuff and operating on whole sections isn't that frequent anyway.

Reply:
Honestly, I would skip the shortcuts. As you said, I don’t see it being used frequently at all to warrant a _shortcut_. I would rather have shortcuts for the main operations, otherwise even figuring out which shortcuts to care about becomes too hard!

Reply:
Agreed!

Reply:
Oh, I see though you are trying to mirror the current cells… maybe we should simplify all shortcuts to be cell below? If you want to introduce it elsewhere… then use the mouse or move the focus up and press the button?

Reply:
Yeah, I am thinking to remove the cell above ones. It is very easy to move the focus up and create a cell below. Also less shortcuts to remember :)

Reply:
> Oh, I see though you are trying to mirror the current cells

Kind of, but we don't have shortcuts for inputs as well, it's relatively rare and is one click away, so it makes sense to stay practical about the number of shortcuts we have.

> maybe we should simplify all shortcuts to be cell below

I don't think it hurts to have the *above* alternatives since they differ only in latter case, and inserting cells is a primary operation, so it can actually be useful. Also consider this example:

```
# ...

## Section 1

Cell 1

## Section 2

Cell 2
```

Inserting above cell 2 is not the same as inserting below cell 1.

Reply:
I see. What do you think then about relying on a prefix instead of casing?

for example, m for markdown below and then am for above? Same for n for Elixir and an for Elixir above?

Reply:
It works, but takes the `a` namespace, which I can imagine us needing in the future :thinking:

Reply:
Also I think it's more natural when the shortcuts work the other way round, for example we have `ej` for evaluate + below. Initially I was considering `nj` and `nk`, but we definitely don't want two keystrokes for the primary operation.

Reply:
I mean, it takes the “a” namespace because we are using it :)

My concern is that S is below but M is above. :) what does “a” do in Vi?

Reply:
> My concern is that S is below but M is above.

To be clear, I'm up for keeping `n`/`N` and `m`/`M`, but dropping the section ones altogether.

> what does “a” do in Vi?

It gets into insert mode one char after the current position ^^

Reply:
I think it is fine to use a then, but it is your call. Move as you prefer!

Reply:
My main concern is the mixup of `action + detail` and `detail + action` shortcuts, so I'd stick to the current ones for now.